README.md

```diff
@@ -1,26 +1,113 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
-Things you may want to cover:
+> A mobile web application where you can manage your budget
 
-* Ruby version
+Home page |
+----------|
+![Screenshot from 2021-12-04 13-28-14](https://user-images.githubusercontent.com/65068771/144707791-ac4b41f3-2af1-4945-8c65-627dc30c6fca.png)
 
-* System dependencies
+  
 
-* Configuration
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Database creation
+## Built With
 
-* Database initialization
+- Ruby on Rails
+- PostgreSQL
 
-* How to run the test suite
+## Video description
+[Loom video](https://loom.com/share/9fb0fa00b7c147ad928599dbcdf16334)
 
-* Services (job queues, cache servers, search engines, etc.)
+## Getting Started
 
-* Deployment instructions
+To get a local copy up and running follow these simple example steps.
 
-* ...
+### Prerequisites
 
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Rails](https://gorails.com/)
 
+### Setup
+
+- Make sure you have Ruby on Rails set up properly on your computer
+- Clone or download this repo on your machine
+
+```sh
+git clone git@github.com:andersonlebon/BudgetApp.git
+~ cd BudgetApp
+
+```
+
+### Development Database
+
+```sh
+# Go Postgres shell user
+ sudo -i -u postgres psql
+```
+
+```sh
+ - rails db:setup
+ and then run
+-  rails db:migrate
+
+```
+
+```sh
+# Create a user with password
+create user "user_name" with password 'user_password';
+
+```
+
+### Install
+
+```sh
+bundle install
+```
+
+### Run
+
+```sh
+rails s
+```
+
+#### If you get an error run
+
+```sh
+bin/rails webpacker:install
+and now run  `rails s`
+```
+
+### Test
+
+```sh
+rspec or rake
+```
+
+## Authors
+
+👤 **Anderson Caleb**
+
+- GitHub: [@Andersonlebon](https://github.com/andersonlebon)
+- Twitter: [@BuyananaCaleb](https://twitter.com/BuyananaCaleb)
+- LinkedIn: [Anderson Caleb](https://www.linkedin.com/in/anderson-caleb-915343209/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Original design idea by Gregoire Vella on [Behance](https://www.behance.net/gregoirevella).
+
+## License
+
+[MIT](./LICENSE)
```

Comment:
- [ ] The Licence link is broken. Please fix that to make the README professional.